server/channels/app/license_test.go

```diff
@@ -1,109 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This entire set of tests was repeated again in platform/license_test.go. That's why I removed this.